charts/platform-reports/README.md

```diff
@@ -0,0 +1,21 @@
+# platform-reports
+
+## Upgrading chart
+
+### To 23.12+
+
+1. Remove `prometheus-node-exporter` DaemonSet before the upgrade since selector labels were changed in child helm chart.
+
+1. Run these commands to update the CRDs before applying the upgrade:
+    ```shell
+    kubectl apply --server-side -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.69.1/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagerconfigs.yaml
```

Comment:
nit: should we include those CRDs into the chart deps?

Reply:
They are included, helm doesn't update crds